xdsl/dialects/experimental/stencil.py

```diff
@@ -468,7 +468,10 @@ def get(
 
         return LoadOp.build(
             operands=[field],
-            attributes=attributes,
+            attributes={
```

Comment:
Can you please remove the unused attributes dict?

Reply:
Thanks, done :)